test/components/plots/scatterPlotTests.ts

```diff
@@ -20,32 +20,39 @@ describe("Plots", () => {
                                   .project("y", yAccessor);
       plot.addDataset(dataset);
       plot.renderTo(svg);
-      var circles = (<any> plot)._renderArea.selectAll("circle");
-      var c1 = d3.select(circles[0][0]);
-      var c2 = d3.select(circles[0][1]);
-      assert.closeTo(parseFloat(c1.attr("cx")), 0, 0.01, "first circle cx is correct");
-      assert.closeTo(parseFloat(c1.attr("cy")), 20, 0.01, "first circle cy is correct");
-      assert.closeTo(parseFloat(c2.attr("cx")), 11, 0.01, "second circle cx is correct");
-      assert.closeTo(parseFloat(c2.attr("cy")), 20, 0.01, "second circle cy is correct");
+      var symbols = plot.getAllSelections();
+      var c1 = d3.select(symbols[0][0]);
+      var c2 = d3.select(symbols[0][1]);
+      var c1Position = d3.transform(c1.attr("transform")).translate;
+      var c2Position = d3.transform(c2.attr("transform")).translate;
+      assert.closeTo(parseFloat(c1Position[0]), 0, 0.01, "first symbol cx is correct");
+      assert.closeTo(parseFloat(c1Position[1]), 20, 0.01, "first symbol cy is correct");
+      assert.closeTo(parseFloat(c2Position[0]), 11, 0.01, "second symbol cx is correct");
+      assert.closeTo(parseFloat(c2Position[1]), 20, 0.01, "second symbol cy is correct");
```

Comment:
It would be less fragile, but I'm unsure about its accuracy.  I'll go try it out